static/styles/subscriptions.css

```diff
@@ -223,7 +223,8 @@
 
     .subscriber_list_container {
         position: relative;
-        max-height: 100%;
+        /* 285px is the compensation for elements above subscribers list. */
+        max-height: calc(95vh - 285px);
```

Comment:
This 285px is going to end up being very fragile; what we should do is extract variables for the individual heights that add up to 285x, like we do in `left-sidebar.css`, and add them together to get 285px.

Reply:
Most of the values themselves are dynamically decided so creating variables for them did not appear useful in the sense that they are not reused as most elements set their heights dynamically using `%`, `vh` units. Also this implementation is not so fragile to changes, as both vh, and calc set the height based on changes to height of device (viewport), so if we later on if 282px is not correct then also there shouldn't be any changes to scroll and stream list behavior.
So, I have given description for subtracted compensation in form of a comment.

Reply:
OK, this is OK to resolve this issue, but long term we probably do want nicer logic here.